optimize/qa/schema-integrity-tests/pom.xml

```diff
@@ -20,14 +20,16 @@
     <dependency>
       <groupId>org.elasticsearch.client</groupId>
       <artifactId>elasticsearch-rest-high-level-client</artifactId>
-      <version>7.17.24</version>
+      <version>${elasticsearch.client.test.version}</version>
```

Comment:
I don't think this is correct. The integrity tests are the only part of the code that are still using elastic search 7, I believe the value you are referencing is an elastic search 8, this will likely lead to the test failing